tests/lib/rules/no-tabs.js

```diff
@@ -23,11 +23,46 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n}",
         "function test(){\n" +
         "  //   sdfdsf \n" +
-        "}",
+        "}"
     ],
     invalid: [
+
+        // Tabs in string/template/regex literals should be reported but not auto-fixed
```

Comment:
Nope. 😄 I will try to implement it. I suppose the replacement string should be `\\t` to avoid expanding it into actual tab again, right?